### DISCUSSION TITLE
只有Bilibili Evolved (Offline)能使用，但附加功能没其他功能

### DISCUSSION THREAD

**Discussion by watn656** [the1812/Bilibili-Evolved#4356] [提问 / Q&A] [answered]
对应功能
附加功能没其他功能

问题描述
附加功能只有顶栏布局，没其他功能，在tampermonkey导入Bilibili-Evolved-2.7.4.zip会显示三个脚本，测试只有Bilibili Evolved (Offline)能显示，其他两不显示

脚本版本
2.7.4(页面显示2.7.3）

浏览器版本
Chrome 版本 115.0.5790.110（正式版本） （64 位）

错误信息
console报错——Error handling response: Error: Blocked a frame with origin...

附加截图
![b3](https://github.com/the1812/Bilibili-Evolved/assets/91203005/69585eba-2ad1-4d66-83f0-3ae4f33bea9f)
![b2](https://github.com/the1812/Bilibili-Evolved/assets/91203005/17fc2a55-c128-4750-bdd1-730d376dc814)
![b1](https://github.com/the1812/Bilibili-Evolved/assets/91203005/783607fc-ae5c-40cc-8adf-b646002ee213)

**Comment by Geeyun-JY3** [comment, marked as answer]
> 在tampermonkey导入Bilibili-Evolved-2.7.4.zip会显示三个脚本

你在干嘛？安装脚本？下了 release 的 zip 然后导入脚本管理器来安装？
先认真看看 README 吧。

**Reply by watn656** [reply]
好的，那个我挂加速器，但下不了正式版，我以为下zip那个导入也是相同的用法。

**Reply by Geeyun-JY3** [reply]
> 好的，那个我挂加速器，但下不了正式版，我以为下zip那个导入也是相同的用法。

网络条件不行用正式版 (jsDelivr 源) 啊。

***

到底谁教的下 release 的 zip 啊，怎么老有人这样安装？
